README.md

```diff
@@ -28,6 +28,7 @@ This project tests all builds against each supported version of Fedora, the late
 1. Integrate libpod into CRI-O to replace its existing container management backend
 1. Further work on the podman pod command
 1. Further improvements on rootless containers
+  * [Shortcompings of Rootless Podman](https://github.com/containers/libpod/blob/master/docs/rootless.md)
```

Comment:
Did you mean to make this a separate '*' list rather than the next number in the above list?
Also s/Shortcompings/Shortcomings/

Reply:
Ok, I think you did, but it didn't resolve well,   Maybe change the '*' to '1" and do one more space in front?

Reply:
Should be a separate heading - we should have a `##` level header and a few lines explaining that this link describes the limitations of rootless containers